app/views/exchanges/hbx_profiles/dry_run_dashboard.html.erb

```diff
@@ -1,128 +1,110 @@
-<div class="container">
-  <h3 class="display-4 text-center my-4 bg-primary text-white p-3">
-    <%= l10n("admin_actions.dry_run.applications.title") %>
-  </h3>
-  <div class="row">
-    <div class="col-md-6">
-      <h3>
-        <strong><%= l10n("admin_actions.dry_run.renewal_benefit_coverage_values") %></strong>
-      </h3>
-      <table class="table table-bordered">
-        <thead>
-          <tr>
-            <th><%= l10n("admin_actions.dry_run.category") %></th>
-            <th><%= l10n("admin_actions.dry_run.value") %></th>
-          </tr>
-        </thead>
-        <tbody>
-          <% benefit_coverage_values.each do |category, value| %>
-            <tr>
-              <td><%= category.humanize %></td>
-              <td><%= value.nil? ? '-' : value %></td>
-            </tr>
+<div class="<%= 'container px-0' unless @bs4 %>">
```

Comment:
@kristinmerbach I am assuming we do not need to feature flag this code as this feature is already released to PROD.

Reply:
We do need it here as the admin code has not been put into prod yet, just broker and consumer flows.